### DISCUSSION TITLE
Unpaired Dexcom One Samsung M51

### DISCUSSION THREAD

**Discussion by GithubCris8** [NightscoutFoundation/xDrip#3028] [Bug reports]
Please help ![IMG-20230818-WA0001.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/611a1f7e-3206-458f-ae55-40e6cbc92134)

![IMG-20230818-WA0002.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/b1b0482c-d350-46a8-b41a-ab1cd0a86b40)

![IMG-20230818-WA0003.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/21353a25-18f7-4bc1-9e5d-6429d8bd0c4a)

![IMG-20230818-WA0004.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/99b24703-0979-47d9-862f-b08c8fe9fb7b)

![IMG-20230818-WA0005.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/980932e0-8c05-45a3-ae8c-e0fe1d3cc751)

**Comment by GithubCris8** [comment]
The transmitter doesn't stay paired, I have to trigger it.

**Comment by Navid200** [comment]
Can you also post a screenshot of the G5/G6/G7/1 Status page?

**Comment by GithubCris8** [comment]
![IMG-20230818-WA0007.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/3989e79c-d3ec-477b-aec4-45c53048601a)

**Reply by Navid200** [reply]
Is there another app, like Dexcom One app, connected to the same transmitter?

**Reply by GithubCris8** [reply]
No

**Reply by Navid200** [reply]
What's the last Android patch you have installed?

**Reply by GithubCris8** [reply]
So this is the full story: we had a transmitter which started to have issues on Samsung A33 (android 13) after that software update which gave issues to the Bluetooth. 

We contacted Dexcom and we tried the same transmitter on Samsung M51 but it didn't work. 

Dexcom sent us a new Dexcom One transmitter which didn't work on Samsung A33 (because of the Bluetooth issue) but it worked well on the Samsung M51 (which has Android 12 and no new updates since last year, because he doesn't get updates anymore).

Now, after the sensor expired on the new transmitter we found out about the new update for the Samsung A33 which promised to fix de Bluetooth issues. We updated A33 but we had these issues with losing bluetooth pairing.

And now after we switched to Samsung M51 (which worked before with the new transmitter) we are still having the same issues of not pairing with Samsung A33. 

 Everytime when we receive the pairing signal with the Bluetooth Xdrip connects and gets its data, but after 5 minutes when the pairing should be finished we get the not paired in Xdrip.

**Reply by GithubCris8** [reply]
I found this on FB group and I feel it's the same issue. I wrote here after that. Maybe it has something to do with the newer transmitters. 

https://m.facebook.com/groups/xDripG5/permalink/9496448937095183/

**Reply by GithubCris8** [reply]
I just want to mention that Samsung M51 is now the main phone which tries to connect to the transmitter. 

Samsung A33 is just a follower, it gets data from Samsung M51. 

There are not 2 different phones which try to connect to the same transmitter but we still have issues.

**Comment by Navid200** [comment]
Follow this to enable detail logging: https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

Restart the phone.
Get it to pair.  Wait for it to unpair.

Do it again.  Get it to pair.  Wait for it to unpair.

Then, upload the logs and post here with an identifying remark you included in the logs.

**Reply by Navid200** [reply]
I forgot, you need to update to the latest Nightly before going through collecting logs for upload:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.08.10

**Reply by GithubCris8** [reply]
Ok. The screenshots I posted here were made using these instructions because I followed other topics. OB1 collection is active. 

But I will do it later today (when my son is back) using the new Xdrip version.

**Comment by GithubCris8** [comment]
After the first unpair with the latest version of xdrip 

![IMG-20230818-WA0009.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/71e2ba71-62ca-4f2d-b1ee-af1d546965b3)

**Reply by GithubCris8** [reply]
![IMG-20230818-WA0010.jpg](https://github.com/NightscoutFoundation/xDrip/assets/142542474/c8ade637-e170-494d-9c75-da794120de24)

**Reply by Navid200** [reply]
You may have misunderstood the instructions.
Please look at them again:
https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

And when you upload, not attach, please also add the following email address to send the logs to:

xDrip.navid at gmail dot com

**Reply by GithubCris8** [reply]
Yes I will send them on that email address. I did it this way because I created this GitHub account on my personal phone, not on my kid's phone. 

But I will send them on that email. Thank you.

**Reply by GithubCris8** [reply]
Hi I sent on xdrip adding your email. The feedback will be here, in this thread ?
